Mathlib/Algebra/Order/Monoid/Unbundled/Pow.lean

```diff
@@ -304,4 +304,10 @@ theorem one_le_zpow {x : G} (H : 1 ≤ x) {n : ℤ} (hn : 0 ≤ n) : 1 ≤ x ^ n
   rw [zpow_natCast]
   apply one_le_pow_of_one_le' H
 
+@[to_additive zsmul_pos]
+lemma one_lt_zpow {x : G} (hx : 1 < x) {n : ℤ} (hn : 0 < n) : 1 < x ^ n := by
```

Comment:
A lemma with the same name is already at [one_lt_zpow](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Order/Group/Basic.html#one_lt_zpow) but with different classes